packages/playground-ui/theme.css

```
@@ -400,4 +400,13 @@ html.light {
       transform: translateX(100%);
     }
   }
+
+  @keyframes shimmer-text {
+    0% {
+      background-position: 200% 0;
+    }
+    100% {
+      background-position: -200% 0;
+    }
```

Stylelint (17.12.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)